docs/source/user/installing.rst

```diff
@@ -17,60 +17,192 @@ Our supported platforms are:
+
+`Berryconda <https://https://github.com/jjhelmus/berryconda>`_ is a
+conda-based Python distribution for the Raspberry Pi.  We are now uploading
+packages to the ``numba`` channel on Anaconda Cloud for ARMv7-based boards,
+which currently incudes the the Raspberry Pi 2 and 3, but not the Pi 1 or
+Zero.  These can be installed using conda from the `numba` channel::
+
+    $ conda install -c numba numba
+
+Berryconda and Numba may work on other Linux-based ARMv7 systems, but this has
+not been tested.
+
+
+Installing on Linux ARMv8 (AArch64) Platforms
+---------------------------------------------
+
+Although many ARMv8 platforms are likely to require building Numba from
+source, we do build and test conda packages on the `NVIDIA Jetson TX2
+<https://www.nvidia.com/en-us/autonomous-machines/embedded-systems-dev-kits-modules/>`_.
+These packages rely on an infrequently updated port of Berryconda to ARMv8
+called `Jetconda <https://github.com/seibert/jetconda/releases>`_.  Jetconda
+is built on Ubuntu 16.04 running on the Jetson TX2, although the packages may
+work on other ARMv8 systems running Ubuntu 16.04.  Once Jetconda is installed,
+Numba can be installed from the ``numba`` channel
+
+    $ conda install -c numba numba
+
+To enable CUDA support on the TX2, set the ``NUMBAPRO`` environment variables
+as described above.
+
 
 .. _numba-source-install-instructions:
 
 Installing from source
 ----------------------
 
-We won't cover requirements in detail here, but you can get the bleeding-edge
-source code from `Github <https://github.com/numba/numba>`_::
+Installing Numba from source is fairly straightforward (similar to other
+Python packages), but installing `llvmlite
+<https://github.com/numba/llvmlite>`_ can be quite challenging due to the need
+for a special LLVM build.  If you are building from source for the purposes of
+Numba development, see :ref:`buildenv` for details on how to create a Numba
+development environment with conda.
 
-   $ git clone git://github.com/numba/numba.git
+If you are building Numba from source for other reasons, first follow the
+`llvmlite installation guide <https://llvmlite.readthedocs.io/en/latest/admin-guide/install.html>`_.
+Once that is completed, you can download the latest Numba source code from 
+`Github <https://github.com/numba/numba>`_::
 
-Source archives of the latest release can be found on
-`PyPI <https://pypi.python.org/pypi/numba/>`_.
+    $ git clone git://github.com/numba/numba.git
 
-You will need a C compiler corresponding to your Python installation, as
-well as the `Numpy <http://www.numpy.org/>`_ and
-`llvmlite <https://github.com/numba/llvmlite>`_ packages.  See :ref:`buildenv`
-for more information.
+Source archives of the latest release can also be found on
+`PyPI <https://pypi.org/project/numba/>`_.  In addition to ``llvmlite``, you will also need:
 
-Checking your installation
---------------------------
+  * A C compiler compatible with your Python installation
```

Comment:
The indenting of the list make it render strangely. Also, want to mention the Anaconda toolchains?